src/handle.rs

```diff
@@ -0,0 +1,20 @@
+// Copyright 2019 Tyler Neely
```

Comment:
Nit: Year is 2020 now 😄

Reply:
But Tyler did the work back in 2019, I think his copyright hold since then.

Reply:
I don't have a preference what happens here